src/main/java/com/studypals/domain/studyManage/entity/StudyCategory.java

```diff
@@ -0,0 +1,61 @@
+package com.studypals.domain.studyManage.entity;
+
+import jakarta.persistence.*;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import com.studypals.domain.memberManage.entity.Member;
+
+/**
+ * 공부 카테고리에 대한 엔티티입니다. JPA 에 의해 관리됩니다.
+ * <p>
+ * 다음과 같은 필드를 가지고 있습니다.
+ * <pre>
+ *     {@code
+ * Long id;
+ * Member member;
+ * Integer dayBelong;
+ * String color;
+ * String description;
+ *     }
+ * </pre>
+ *
+ * <p><b>주요 생성자:</b><br>
+ * {@code Builder}  <br>
+ * builder 패턴을 통해 생성합니다. <br>
+ *
+ * @author jack8
+ * @since 2025-04-10
+ */
+@Entity
+@Builder
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
[P3]

최근에 프로젝트 정리하며 알게 된 사실인데, 이전 프로젝트에선 해당 생성자의 접근 권한을 `protected`로 했던 것을 발견했습니다.
그때 왜 그렇게 했는지 다시 찾아보니 기본 생성자의 사용을 제한함으로써 캡슐화를 지키고, 객체의 무분별한 생성을 막을 수 있다는 점에서 접근 권한을 제한한다고 합니다.
이때 `private`이 아닌 `protected`인 이유는 JPA가 기본 생성자를 통해 프록시 객체를 생성해 사용하기 때문이라고 합니다.

이것도 지난번 멤버 관련 PR에서 놓쳤던 부분인데 지금이라도 코멘트 남겨봅니다.

```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```

Reply:
좋네요~ 수정했습니다. 
JwtToken ,Member ,StudyCategory, StudyTime 의 NoArgsConstructor을 protected로 변경하였습니다.

Reply:
반영 감사합니다~!